core/server/web/members/app.js

```diff
@@ -39,8 +39,8 @@ module.exports = function setupMembersApp() {
     membersApp.put('/api/subscriptions/:id', (req, res, next) => membersService.api.middleware.updateSubscription(req, res, next));
 
     // API error handling
-    membersApp.use(shared.middlewares.errorHandler.resourceNotFound);
-    membersApp.use(shared.middlewares.errorHandler.handleJSONResponseV2);
+    membersApp.use('/api', shared.middlewares.errorHandler.resourceNotFound);
+    membersApp.use('/api', shared.middlewares.errorHandler.handleJSONResponseV2);
```

Comment:
I think moving these we lose error handling for the webhook endpoint - should I add it to there too? :thinking:

Reply:
done this now